App/navigation/AppTabNavigation.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import {HomeScreen, VideoScreen, RadioScreen, ScheduleScreen} from '../screens';
+import {Colors, Routes, Icons} from '../constants';
+import FontTelloIcon from '../components/FontTelloIcon';
+
+const MyTabNavigator = () => {
+  const Tab = createBottomTabNavigator();
+  const IconSize = 30;
```

Comment:
put this in constants and make diff sizes like iconSmall = 20, iconLarge = 40 etc